### ISSUE TITLE
Many things in Editor screen are not operational through a keyboard (Tab) interface.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#813]
**Describe the bug**
Many things in Editor screen are not operational through keyboard standard keys. If we want to access different tabs on the Editor page such as "Layout" and "Style" Tab then we are not able to select them through keyboard. Also various menus such as "Action", "View", "Publishing" etc in the Content Tab are not accessible through keyboard.

**To Reproduce**
1. Go to Editor screen and select any page for a site and edit it.
2. See that all the options/controls on the screen are not accessible using standard Keyboard keys like Tab key.

**Expected behavior**
Whole page should be accessible using Standard keyboard Keys.

**Comment by ramdev1980** [comment]
Similar bug is already raised ( Tab sequence in Editor screen issue #794 ). Closing it as it is duplicated.
